### PR TITLE
Adding Moonbeam network support.

### PR DESCRIPTION
**Pull request by m00nbeans**
Adds support for the recently launched EVM-compatible Polkadot parachain, Moonbeam. The sister network to Moonriver, which is already supported by UseDapp.

### CODE REVIEW

**Comment by m00nbeans** [comment]
@marekkirejczyk would you mind taking a look at this when you get a chance? Thank you 😁

File: .changeset/twelve-dancers-leave.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@usedapp/core': patch
+---
+
+Adding the moonbeam network.
```
Comment (jodator) [inline .changeset/twelve-dancers-leave.md:5]:
```suggestion
Add the moonbeam network
```

FYI we follow [this convention](https://cbea.ms/git-commit/).

**Review by jodator — DISMISSED** [review]
@m00nbeans Thanks for the PR :)

As for the review: Please check the linter settings

Since the contract code is not verified [on the explorer](https://blockscout.moonbeam.network/address/0x47152C4dCE75C77Bc9E52F5AAa2a20117971C365/contracts) I'll wait for someone to double-check the multi call address.

**Comment by tt-marek** [comment]
@m00nbeans Looks good! Would you fix linter and we can merge.

**Comment by m00nbeans** [comment]
@tt-marek @jodator Made the recommended change and fixed the linter - also verified the multicall contract here: https://blockscout.moonbeam.network/address/0x47152C4dCE75C77Bc9E52F5AAa2a20117971C365/contracts

**Review by tt-marek — APPROVED** [review]

**Review by tt-marek — APPROVED** [review]

**Pull request merged**